src/types/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface TodayItemData {
+  status: number;
+  data: {
+    todayProductList: {
+      productId: number;
+      productName: string;
+      productImage: string;
+      discount: number;
+      originalPrice: number;
+      discountedPrice: number;
+      reviewCount: number;
+    };
+  };
+}
+
+export interface UserData {
+  status: number;
+  getMyCoupangUser: {
+    userId: number;
+    profileImage: string;
+    userName: string;
+    payMoney: number;
+    cash: number;
+    reviewCount: number;
+    likeCount: number;
+    recentSeeCount: number;
+  };
+}
```

Comment:
```suggestion
export interface UserData {
  status: number;
  getMyCoupangUser: UserDataprops;
}

export interface UserDataprops {
  userId: number;
  profileImage: string;
  userName: string;
  payMoney: number;
  cash: number;
  reviewCount: number;
  likeCount: number;
  recentSeeCount: number;
}
```

<br/>

이런 식으로 인터페이스를 나눠주고 사용하고자 하는 컴포넌트에 `import` 하면 any 를 쓰지 않고 타입 지정을 할 수 있을 것 같아!